modules/iopcore/cdvdman/cdvdman.c

```diff
@@ -33,6 +38,11 @@ static void cdvdman_startThreads(void);
 static void cdvdman_create_semaphores(void);
 static int cdvdman_read(u32 lsn, u32 sectors, void *buf);
 
+// Sector cache to improve IO
+#define MAX_SECTOR_CACHE 16
+static u8 sector_cache[MAX_SECTOR_CACHE][2048];
```

Comment:
This is 64KiB of extra "ingame" memory consumption.

Also note in cdvdman.c line55 there is also a sector buffer of 4KiB.

Reply:
This is absolutely necessary if we want to have decent USB speeds.
We might want to test with different sizes (32 sectors worked just fine except with smb, maybe 8 is enough).